src/component/flights/flights.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import './flights.css'
+import { Link } from 'react-router-dom';
+
+const style = {
+  marginLeft: "18px",
+  fontSize: "20px",
+  color: "white",
+  fontWeight: "bold"
+};
+const rightstyle = {
+  marginLeft: "18px",
+  fontSize: "20px",
+  color: "white",
+  textAlign: "right"
+};
+
+const button = {
+  backgroundColor: "#3C5E83",
+  border: "none",
+  color: "white",
+  padding: "6px 25px",
+  textAlign: "center",
+  display: "inline-block",
+  marginLeft: "390px",
+  borderRadius: "5px",
+  fontSize: "20px"
+};
+
+const header = {
+  textAlign: "center",
+  marginLeft: "18px",
+  fontSize: "40px",
+  color: "white"
+};
+
+export const Flights = () => {
+
+  return (
+    <React.Fragment>
+      <section class="custom-form-container">
+
+        <h2 style={header}>Book a Flight</h2>
+        <br></br>
+        <form>
+
+          <div class="form-group row">
+            <label for="cardType" class="col-sm-2 col-form-label" style={style}>Flight Type :</label>
+            <div class="custom-control custom-radio custom-control-inline">
+              <input type="radio" id="customRadioInline1" name="customRadioInline1" class="custom-control-input" />
+              <label class="custom-control-label" for="customRadioInline1" style={style}>
+                <img width="60px" height="60px" src="https://img.icons8.com/dusk/64/000000/circular-arrows.png" />
+              </label>
+            </div>
+            <div class="custom-control custom-radio custom-control-inline">
+              <input type="radio" id="customRadioInline2" name="customRadioInline1" class="custom-control-input" />
+              <label class="custom-control-label" for="customRadioInline2"></label>
+              <img width="60px" height="60px" src="https://img.icons8.com/doodle/48/000000/one-way-road-sign.png"></img>
+            </div>
+          </div>
+
```

Comment:
I would suggest we try using a grid to place these elements on the page.

Reply:
okay I can take a look at that and see if I can place it on a grid